src/main/java/com/wcc/platform/controller/DefaultController.java

```diff
@@ -30,10 +28,22 @@ public FooterController(final CmsService cmsService) {
    *
    * @return Footer content.
    */
-  @GetMapping("/footer")
+  @GetMapping("/api/cms/v1/footer")
   @Operation(summary = "API to retrieve footer section details")
   @ResponseStatus(HttpStatus.OK)
   public ResponseEntity<FooterPage> getFooterPage() {
     return ResponseEntity.ok(cmsService.getFooter());
   }
+
+  /**
+   * Create default api response for WCC Platform
+   *
+   * @return "Women Coding Community Platform"
+   */
+  @GetMapping("/")
+  @Operation(summary = "API to retrieve default page")
+  @ResponseStatus(HttpStatus.OK)
+  public ResponseEntity<String> helloWorld() {
```

Comment:
I think this will be replaced with HomePage content ? can we rename this to homePage ?

Reply:
This was for testing, I think homePage should follow the api structure `/api/cms/v1/landingPage`

Reply:
I use that for testing, actually I am going to remove now.